jupyterlite_sphinx/jupyterlite_sphinx.css

```
@@ -75,3 +76,25 @@
     display: none;
   }
 }
+
+/* Some improvements for hidden buttons */
+
+/* smooth transitions */
+.try_examples_button {
+  opacity: 1;
+  transition: opacity 0.3s ease-in-out;
+}
+
+.try_examples_button.hidden {
+  display: none;
+  opacity: 0;
+}
+
+.try_examples_button.fade-in {
+  transition: opacity 0.3s ease-in-out;
+}
+
+/* to ensure hidden class takes precedence */
+.hidden {
+  display: none !important;
+}
```

I'm not entirely sure that it's worth creating the fade-in effect when there is also a content layout shift.  

But if we still want to keep it, I suggest a few changes:

```suggestion

/* Buttons are hidden by default and revealed via JavaScript in another file */

/* The button is revealed in two stages. The first stage removes the hidden class.
  This gives the button dimensions and causes the page content below to "jump" 
  (get pushed down suddenly). The second stage removes the transparent class,
  which reveals the button with a slight fade-in effect. */
.try_examples_button.transparent {
  opacity: 0;
}

.try_examples_button {
  opacity: 1;
  transition: opacity 0.3s ease-in-out;
}
```